packages/design-system-react-native/scripts/build.js

```diff
@@ -10,20 +10,8 @@ try {
   console.log('Step 3: Building the project...');
   execSync('tsc --project tsconfig.build.json', { stdio: 'inherit' });
 
-  console.log('Step 4: Copying non-TypeScript files...');
-
-  // Copy Blockies utilities
-  // TODO: Remove this step when blockies is converted to typescript
-  execSync('mkdir -p dist/components/temp-components/Blockies', {
-    stdio: 'inherit',
-  });
-  execSync(
-    'cp src/components/temp-components/Blockies/Blockies.utilities.js dist/components/temp-components/Blockies/',
-    { stdio: 'inherit' },
-  );
-
   // Copy SVG files
-  console.log('Step 5: Copying SVG assets...');
+  console.log('Step 4: Copying SVG assets...');
```

Comment:
Updating the build script now that the TypeScript version of the Blockies utility is included in the build via the TypeScript compiler.